phoenix-core/src/it/java/org/apache/phoenix/end2end/index/IndexExpressionIT.java

```diff
@@ -123,7 +126,7 @@ protected void helpTestCreateAndUpdate(boolean mutable, boolean localIndex) thro
                     + (localIndex ? "LOCAL" : "")
                     + " INDEX IDX ON "
                     + fullDataTableName
-                    + " ((UPPER(varchar_pk) || '_' || UPPER(char_pk) || '_' || UPPER(varchar_col1) || '_' || UPPER(char_col2)),"
+                    + " ((UPPER(varchar_pk) || '_' || UPPER(char_pk) || '_' || UPPER(varchar_col1) || '_' || UPPER(b.char_col2)),"
```

Comment:
Just curious about the prefixing of the column name with "b". Was that required, or were you just testing that this wouldn't be a problem?